### PR TITLE
add chartjs-chart-pcp

### PR DESCRIPTION
**Pull request by sgratzl**
add link to https://github.com/sgratzl/chartjs-chart-pcp

Awesome Contribution Checklist:

- [x] I have read, and re-read the [Contributing Guidelines](https://github.com/chartjs/awesome/blob/master/CONTRIBUTING.md)
- [x] I have searched to ensure the suggested item doesn't exist on this list
- [x] This PR contains only one item

### Please Describe Your Addition

add link to https://github.com/sgratzl/chartjs-chart-pcp for showing parallel coordinate plots (PCPs).

image

### CODE REVIEW

**Review by benmccann — DISMISSED** [review]
Cool! Thanks so much for sharing!

My only suggestion is that it'd be really helpful to host an interactive version on github pages or someplace like codepen or plunkr

Also, I noticed the samples use `Chart.bundle.js`. You probably can drop the `bundle` portion of the fine name because I believe that bundled `moment` for time charts

**Comment by sgratzl** [comment]
> My only suggestion is that it'd be really helpful to host an interactive version on github pages or someplace like codepen or plunkr

take a look at the README. There are Codepen links below each example, e.g. https://codepen.io/sgratzl/pen/OJyyoOw

File: README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ A curated list of awesome things related to [Chart.js](https://www.chartjs.org)
 - [financial](https://github.com/chartjs/chartjs-chart-financial) - Adds financial chart types such as a candlestick.
 - [geo](https://github.com/sgratzl/chartjs-chart-geo) - Adds geographic map chart types such as choropleth and bubble map.
 - [graph](https://github.com/sgratzl/chartjs-chart-graph) - Adds graph chart types such as a force directed graph.
+- [pcp](https://github.com/sgratzl/chartjs-chart-pcp) - Adds parallel coordinates plot chart type.
```
Comment (etimberg) [inline README.md]: Can we put this alphabetically after `matrix`?

Comment (sgratzl) [reply README.md]: my bad - fixed

**Review by kurkle — APPROVED** [review]

**Review by benmccann — APPROVED** [review]

**Review by etimberg — APPROVED** [review]

**Pull request merged**